aspnetcore/host-and-deploy/aspnet-core-module.md

```diff
@@ -1,37 +1,49 @@
 ---
-# ASP.NET Core Module configuration reference
+# ASP.NET Core Module
 
-By [Luke Latham](https://github.com/guardrex), [Rick Anderson](https://twitter.com/RickAndMSFT), [Sourabh Shirhatti](https://twitter.com/sshirhatti), and [Justin Kotalik](https://github.com/jkotalik)
-
-This document provides instructions on how to configure the ASP.NET Core Module for hosting ASP.NET Core apps. For an introduction to the ASP.NET Core Module and installation instructions, see the [ASP.NET Core Module overview](xref:fundamentals/servers/aspnet-core-module).
+By [Tom Dykstra](https://github.com/tdykstra), [Rick Strahl](https://github.com/RickStrahl), [Chris Ross](https://github.com/Tratcher), [Rick Anderson](https://twitter.com/RickAndMSFT), [Sourabh Shirhatti](https://twitter.com/sshirhatti), [Justin Kotalik](https://github.com/jkotalik), and [Luke Latham](https://github.com/guardrex)
 
 ::: moniker range=">= aspnetcore-2.2"
 
-## Hosting model
+The ASP.NET Core Module is a native IIS module that plugs into the IIS pipeline to either:
+
+* Host an ASP.NET Core app inside of the IIS worker process (`w3wp.exe`), called the [in-process hosting model](#in-process-hosting-model).
+* Forward web requests to a backend ASP.NET Core app running the [Kestrel server](xref:fundamentals/servers/kestrel), called the [out-of-process hosting model](#out-of-process-hosting-model).
+
+Supported Windows versions:
+
+* Windows 7 or later
+* Windows Server 2008 R2 or later
+
+When hosting in-process, the module uses an in-process server implementation for IIS, called IIS HTTP Server (`IISHttpServer`).
 
-For apps running on .NET Core 2.2 or later, the module supports an in-process hosting model for improved performance compared to reverse-proxy (out-of-process) hosting. For more information, see <xref:fundamentals/servers/aspnet-core-module#aspnet-core-module-description>.
+When hosting out-of-process, the module only works with Kestrel. The module is incompatible with [HTTP.sys](xref:fundamentals/servers/httpsys).
 
-In-procsess hosting is opt-in for existing apps, but [dotnet new](/dotnet/core/tools/dotnet-new) templates default to the in-process hosting model for all IIS and IIS Express scenarios.
+## Hosting models
 
-To configure an app for in-process hosting, add the `<AspNetCoreHostingModel>` property to the app's project file (for example, *MyApp.csproj*) with a value of `InProcess` (out-of-process hosting is set with `outofprocess`):
+### In-process hosting model
+
+To configure an app for in-process hosting, add the `<AspNetCoreHostingModel>` property to the app's project file with a value of `InProcess` (out-of-process hosting is set with `OutOfProcess`):
 
 ```xml
 <PropertyGroup>
   <AspNetCoreHostingModel>InProcess</AspNetCoreHostingModel>
 </PropertyGroup>
 ```
 
+If the `<AspNetCoreHostingModel>` property isn't present in the file, the default value is `OutOfProcess`.
+
 The following characteristics apply when hosting in-process:
```

Comment:
"characteristics" seem to fit more in the conceptual overview doc than in this configuration doc.

Reply:
These are *implementation characteristics*. I don't think these are "fundamental" concepts ... Do readers really need to be bogged down in these details when they're trying to get their bearings?

* requestTimeout
* Use of app_offline.htm
* bitness issue
* host setup
* client disconnects
* working directory

Except for the images and a few sentences, I still cringe a bit at moving as much back to *Servers* as we just did ... however ...

The current split of content back into Servers doesn't seem to require the reader to go back to that topic for a successful deployment. Therefore, I don't think that links back out to that topic are currently required. If these characteristics are moved out, then I'll need to put links in at the top, and readers will need to go back out to get these details. We'd be splitting implementation details across two topics.

Anywho ... just my 2&cent; ... let me know what you want to do.

Reply:
Feel free to follow your own inclination.  At this point you're more familiar than I am with the new material and how everything fits together.